packages/internal-tooling/package.json

```diff
@@ -24,8 +24,11 @@
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-react-hooks": "^1.6.0",
+    "fela": "^10.6.1",
```

Comment:
We should wait for #1743 to be merged first and remove this change here.
@layershifter Do you agree?